modules/telco-troubleshooting-certs-auto-node.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-cert-maintenance.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="telco-troubleshooting-certs-auto-node_{context}"]
+= Node certificates
+
+Node certificates are self-signed certificates, which means that they are signed by the cluster and they originate from an internal certificate authority (CA) that is generated by the bootstrap process.
+
+After the cluster is installed, the cluster automatically renews the node certificates.
+
+For more information, see "Node certificates" in _Security and compliance_.
```

Comment:
also in the list of Additional resources in the first section; maybe that's by design (ignore if so)